doc-content/enterprise-only/openshift/hacep-server-maven-proc.adoc

```diff
@@ -0,0 +1,132 @@
+[id='hacep-server-maven-proc']
+= Implementing the HA CEP server with a Maven repository for updating the KJAR service
+
+You can implement the HA CEP server that retrieves the KJAR service and all dependencies from a Maven repository that you provide. In this case, you can update the KJAR service at any time by updating it in the Maven repository and then updating the group, artifact, and version (GAV) values in an environment variable of the server pod.
```

Comment:
The part "and then updating the group, artifact, and version (GAV) values in an environment variable of the server pod." isn't correct and must be removed